docs/source/shaders/material/as_toon.rst

```diff
@@ -350,19 +350,19 @@ Contour Parameters
     Generate the contours based on the object ID.
 
 *Contour Material*
-    Generate contours based on the material ID. Objects with the same material ID would share contours.
+    Generate contours based on the material ID. Objects with the same material ID will share contours.
 
 *Contour Occlusion*
-    Generate contours based on ambient occlusion.
+    Generate contours based on ambient-occlusion.
 
 *Contour Creases*
-    Generate contours based on the creases the object [#]_, outlining sudden changes of the surface.
+    Generate contours based on the creases of the object [#]_, outlining sudden changes of the surface.
 
 *Occlusion Threshold*
-    The threshold value for the ambient occlusion based outlining. Low values generate outlines in shorter ray hit distances.
+    The threshold value for the ambient-occlusion based outlining. Low values generate outlines in shorter ray hit distances.
```

Comment:
Revert.

Reply:
The occlusion threshold is used when comparing depths to decide if a contour is present ot not.